content/en/docs/topics/scoping/index.md

```diff
@@ -135,10 +135,10 @@ To ignore other elements, add the elements of your choice under `IgnoredScopes`
 ```ini
 StylesPath = .github/styles
```

Comment:
I've seen this being used with a placeholder `"..."`. Feel free to change the path if you think it might cause confusion.